src/NuGet.Core/NuGet.Common/ConcurrencyUtilities.cs

```diff
@@ -197,7 +197,9 @@ private static FileStream AcquireFileStream(string lockPath)
                 options: options);
         }
 
+#pragma warning disable IDE1006 // Naming Styles
         private static string _basePath;
+#pragma warning restore IDE1006 // Naming Styles
```

Comment:
I would undo this change.  I don't see a warning/error.  If there is indeed a warning/error, let's figure it out.  If the problem is specific to me, then I'd want to think about other options (e.g.:  modifying or disabling the rule) instead of disabling it selectively.

Reply:
Fixed on my end. 

The editor config seems right.